app/models/miq_region.rb

```diff
@@ -18,6 +18,8 @@ class MiqRegion < ApplicationRecord
   virtual_has_many :miq_templates,          :uses => :all_relationships
   virtual_has_many :vms,                    :uses => :all_relationships
 
+  after_save { MiqRegion.my_region_clear_cache }
```

Comment:
Prefer creating a private method and referring to it here.  This is the pattern we use in the other models, and also, there used to be a problem when specifying blocks to after_save where it would lose other callbacks values (not sure that still applies, but battle scars).